2023-09-20 19:40:26,927 -- ERROR -- Internal Server Error: / --> C:\Users\Constantin\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\utils\log.py *** (<class 'django.urls.exceptions.NoReverseMatch'>, NoReverseMatch("Reverse for 'test' not found. 'test' is not a valid view function or pattern name."), <traceback object at 0x0000029420E48380>)
Traceback (most recent call last):
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\utils\deprecation.py", line 138, in __call__
    response = self.process_response(request, response)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\contrib\flatpages\middleware.py", line 12, in process_response
    return flatpage(request, request.path_info)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\contrib\flatpages\views.py", line 45, in flatpage
    return render_flatpage(request, f)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\utils\decorators.py", line 133, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\contrib\flatpages\views.py", line 70, in render_flatpage
    return HttpResponse(template.render({"flatpage": f}, request))
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\backends\django.py", line 62, in render
    return self.template.render(context)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\loader_tags.py", line 54, in render
    result = self.nodelist.render(context)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\defaulttags.py", line 472, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\urls\resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'test' not found. 'test' is not a valid view function or pattern name.
2023-09-20 19:40:26,943 -- ERROR -- "GET / HTTP/1.1" 500 146913 --> C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\core\servers\basehttp.py *** None
2023-09-20 19:43:59,418 -- ERROR -- Internal Server Error: /news/test/ --> C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\utils\log.py *** None
2023-09-20 19:44:00,658 -- ERROR -- "GET /news/test/ HTTP/1.1" 500 0 --> C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\core\servers\basehttp.py *** None
2023-09-20 19:59:31,621 -- ERROR -- Internal Server Error: /news/test/ --> C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\utils\log.py *** None
2023-09-20 19:59:31,622 -- ERROR -- "GET /news/test/ HTTP/1.1" 500 0 --> C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\core\servers\basehttp.py *** None
2023-09-20 20:01:26,886 -- ERROR -- Internal Server Error: /news/test/ --> C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\utils\log.py *** None
2023-09-20 20:01:26,887 -- ERROR -- "GET /news/test/ HTTP/1.1" 500 1 --> C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\core\servers\basehttp.py *** None
2023-09-20 20:02:37,535 -- ERROR -- Internal Server Error: /news/test/ --> C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\utils\log.py *** None
2023-09-20 20:02:37,536 -- ERROR -- "GET /news/test/ HTTP/1.1" 500 3136 --> C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\core\servers\basehttp.py *** None
2023-09-20 20:02:44,671 -- ERROR -- Internal Server Error: /news/test/ --> C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\utils\log.py *** None
2023-09-20 20:02:44,672 -- ERROR -- "GET /news/test/ HTTP/1.1" 500 3136 --> C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\core\servers\basehttp.py *** None
2023-09-20 20:04:27,721 -- ERROR -- Internal Server Error: /news/ --> C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\utils\log.py *** (<class 'django.urls.exceptions.NoReverseMatch'>, NoReverseMatch("Reverse for 'test' not found. 'test' is not a valid view function or pattern name."), <traceback object at 0x000001D02F8D1A00>)
Traceback (most recent call last):
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\backends\django.py", line 62, in render
    return self.template.render(context)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\defaulttags.py", line 472, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\urls\resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'test' not found. 'test' is not a valid view function or pattern name.
2023-09-20 20:04:27,728 -- ERROR -- "GET /news/ HTTP/1.1" 500 219339 --> C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\core\servers\basehttp.py *** None
2023-09-20 20:04:56,511 -- ERROR -- Internal Server Error: /news/ --> C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\utils\log.py *** (<class 'django.urls.exceptions.NoReverseMatch'>, NoReverseMatch("Reverse for 'test' not found. 'test' is not a valid view function or pattern name."), <traceback object at 0x000001F84340DF80>)
Traceback (most recent call last):
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\backends\django.py", line 62, in render
    return self.template.render(context)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\defaulttags.py", line 472, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\urls\resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'test' not found. 'test' is not a valid view function or pattern name.
2023-09-20 20:04:56,517 -- ERROR -- "GET /news/ HTTP/1.1" 500 219334 --> C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\core\servers\basehttp.py *** None
2023-09-20 20:07:32,525 -- ERROR -- Internal Server Error: /news/ --> C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\utils\log.py *** (<class 'django.urls.exceptions.NoReverseMatch'>, NoReverseMatch("Reverse for 'test' not found. 'test' is not a valid view function or pattern name."), <traceback object at 0x0000016ED51BCA00>)
Traceback (most recent call last):
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\backends\django.py", line 62, in render
    return self.template.render(context)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\defaulttags.py", line 472, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\urls\resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'test' not found. 'test' is not a valid view function or pattern name.
2023-09-20 20:07:32,532 -- ERROR -- "GET /news/ HTTP/1.1" 500 219471 --> C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\core\servers\basehttp.py *** None
2023-09-20 20:07:38,554 -- ERROR -- Internal Server Error: /news/ --> C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\utils\log.py *** (<class 'django.urls.exceptions.NoReverseMatch'>, NoReverseMatch("Reverse for 'test' not found. 'test' is not a valid view function or pattern name."), <traceback object at 0x0000016ED52335C0>)
Traceback (most recent call last):
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\backends\django.py", line 62, in render
    return self.template.render(context)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\template\defaulttags.py", line 472, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\urls\resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'test' not found. 'test' is not a valid view function or pattern name.
2023-09-20 20:07:38,560 -- ERROR -- "GET /news/ HTTP/1.1" 500 219334 --> C:\Users\CORSAIR_90\PycharmProjects\Task_NewsPortal\venv\lib\site-packages\django\core\servers\basehttp.py *** None
